### PR TITLE
Fixes #33522 - Create Ansible config job

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by xprazak2** [comment]
Blocked by https://github.com/theforeman/foreman_ansible/pull/419

**Comment by xprazak2** [comment]
Rebased.

File: webpack/components/AnsibleHostDetail/components/JobsTab/NewRecurringJobHelper.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    jobInvocation: {
+      hostIds: [hostId],
+      feature: 'ansible_run_host',
+      targetingType: 'static_query',
+      scheduling: {
+        startAt: date,
+      },
+      recurrence: {
+        cronLine: toCron(date, repeat),
+        purpose: `ansible-host-${hostId}`,
```
Comment (ezr-ondrej) [inline webpack/components/AnsibleHostDetail/components/JobsTab/NewRecurringJobHelper.js:56]: Could we have a function, that would return the purpose, I see it quite dangerous to have it in two places and rely on them being changed at the same time.

**Review by ezr-ondrej — COMMENTED** [review]
Works as expected, just one nit if we could extract the `purpose` into a helper function.

File: webpack/components/AnsibleHostDetail/components/JobsTab/NewRecurringJobHelper.js (hunk 1 above)
Comment (xprazak2) [reply webpack/components/AnsibleHostDetail/components/JobsTab/NewRecurringJobHelper.js]: I agree, updated.

**Comment by ezr-ondrej** [comment]
Now linter is barking about something....

**Review by ezr-ondrej — APPROVED** [review]
Thanks @xprazak2 ! :+1:

**Pull request merged**